Rakefile

```diff
@@ -104,3 +104,9 @@ task :release, :version do |task, args|
                    name: version,
                    body: changelog.split(/^# /)[2].strip)
 end
+
+desc 'Runs pod install on the Demo project, used for testing too.'
+task :pod_install do
+  sh "brew install xcproj" unless `which xcproj`.strip.length > 0
+  sh "cd Demo ; COCOAPODS_DISABLE_DETERMINISTIC_UUIDS=true bundle exec pod install"
```

Comment:
This moved into the Podfile: https://guides.cocoapods.org/syntax/podfile.html#install_bang

So this wasn't doing anything, `install! 'cocoapods', :deterministic_uuids => false` if you need it. You may not need it.